.circleci/config.yml

```diff
@@ -422,3 +431,28 @@ jobs:
               --self-contained-html \
               internal/buildscripts/packaging/tests/installer_test.py
       - save_pytest_results
+
+  windows-msi:
+    executor:
+      name: win/default
+      shell: powershell.exe
+    steps:
+      - attach_to_workspace
+      - run:
+          command: mkdir -p dist
+      - run:
+          name: Install Wix Toolset
+          command: .\internal\buildscripts\packaging\msi\make.ps1 Install-Tools
+      - run:
+          name: Build MSI
+          command: |
+            $Version = if ($env:CIRCLE_TAG -match '^v(\d+\.\d+\.\d+)') { $Matches[1] } else { "0.0.1" }
```

Comment:
So non-released builds will get version 0.0.1 it looks like. Does this impact being able to upgrade them? If so should the build number be added to it so it increments?